### ISSUE TITLE
[FEATURE] Add a sound when a trade is found

### ISSUE THREAD

**Issue by Chuckels745** [Greeenman999/LibrarianTradeFinder#43]
When a trade is found a sound effect is played (could be turned on and off in the config menu). Could be the advancements sound or smoothing else idk.  Could be cool

**Comment by Greeenman999** [comment]
Thank you for your suggestion. I'm going to add this to my todo list ;D

**Comment by dgra7** [comment]
I agree a bell sound or something similiar when the trade is found would be really nice, even better if you can auto-lock the first found trade so the villager doesn't change it.
